modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -40,6 +40,14 @@ A user requires the following privileges to install an {product-title} cluster:
 
 For more information about creating an account with only the required privileges, see link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-5372F580-5C23-4E9C-8A4E-EF1B4DD9033E.html[vSphere Permissions and User Management Tasks] in the vSphere documentation.
 
+[discrete]
+[id="installation-vsphere-installer-infra-requirements-vmotion_{context}"]
+== Using {product-title} in vMotion-enabled vSphere clusters
+
+{product-title} only supports compute-only vMotion in vMotion-enabled clusters. If you are using vSphere volumes in your pods, migrating a VM across datastores either manually or through Storage vMotion causes invalid references within {product-title} persistent volume (PV) objects. This prevents affected pods from starting up and can result in data loss. Therefore, any form of Storage vMotion is unsupported.
```

Comment:
"{product-title} only supports compute-only vMotion in vMotion-enabled clusters."

What are you trying to say here? If the person is a vSphere admin reading this it sounds funny.

Reply:
"in vMotion-enabled clusters" to me is redundant. vMotion is not enabled at the cluster level. It is configured per ESXi host.

Reply:
Trying to be more direct than saying "should generally work". How about this? "{product-title}

Reply:
Maybe I am being too picky (you decide) but you do not need a cluster to perform either types of vmotion. vmotion just needs a vmkernel adapter configured per ESXi host and storage vmotion needs nothing.

To enable DRS and Storage DRS yes a vCenter cluster is required but the underlying features do not.

Should we emphasize that storage vmotion is not supported. It is at the end of the paragraph that was just skimmed ;-)

Reply:
I agree that it should be emphasized upfront. And thanks for the clarification about not needing a cluster. How about updating the title and a single paragraph at the beginning:

> **OpenShift Container Platform with vMotion**
> OpenShift Container Platform generally supports compute-only vMotion but not Storage vMotion.
> If you are using vSphere volumes in your pods, migrating a VM...

Reply:
@bobfuru yeah that sounds better

Reply:
OK, updated.